compiler/test/compilable/type_seq_qual.d

```diff
@@ -0,0 +1,16 @@
+alias AliasSeq(Args...) = Args;
+
+immutable(AliasSeq!(int, float)) c;
+immutable AliasSeq!(int, float) d;
+static assert(is(typeof(c) == typeof(d)));
+
+alias TS = AliasSeq!(int, float);
+static assert(is(typeof(c) == immutable TS));
+static assert(is(immutable(TS)[0] == immutable int));
+
+static assert(is(const(TS) == const TS));
+static assert(is(const(AliasSeq!(int, float))[0] == const int)); // OK
```

Comment:
this line actually fails too:
```
compilable/type_seq_qual.d(12): Error: static assert:  `is(const(int) == const(int))` is false
```

Reply:
tsrc.addSTC(STC.const_) -> probably should follow similar to tsrc.immutableOf() by using  tsrc.constOf()